backend/tests/prisonerQuickLook.test.ts

```diff
@@ -90,6 +136,8 @@ describe('prisoner profile quick look', () => {
 
     logError = jest.fn()
 
+    config.apis.nonAssociations.ui_url = nonAssociationsUrl
```

Comment:
does this potentially need a clean-up `afterEach/All` step for other tests? doesn't appear matter now for the current set of tests, but there's no undo is there?

Reply:
Didn't see any similar cleanup blocks in the rest of the file, so I think it's OK, at least until we see it's actually leaking state and causing problems.